util/util.go

```diff
@@ -0,0 +1,44 @@
+package util
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math/big"
+)
+
+// GenerateSalts generates `numKeys` salts with length `lenSalt`.
+func GenerateSalts(numKeys, lenSalt int) (salts [][]byte) {
+	if lenSalt < 8 {
+		fmt.Println("Error in generating the salts: lenSalt must be at leat 8")
+	}
+	salts = make([][]byte, numKeys)
+	for i := 0; i < numKeys; i++ {
+		salts[i] = make([]byte, lenSalt)
+		_, err := rand.Read(salts[i])
+		if err != nil {
+			fmt.Println("Error in generating the salts: ", err)
```

Comment:
It's better practice to always return the error, and let the highest level possible deal with it.  This function can't really be re-used as a library function since it just prints out the errors.